<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterUnsharpMask</class>
 <widget class="QDialog" name="FilterUnsharpMask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>虚光蒙版</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>半径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_radius">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="pageStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_radius">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>数量：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_amount">
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_amount">
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>阈值：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_threshold">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_threshold">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton_reset">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_radius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_radius</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_threshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_threshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_amount</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_amount</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_radius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_radius</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_threshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_threshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_amount</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_amount</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>FilterUnsharpMask</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>FilterUnsharpMask</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
